client/src/pages/lessons/edit.tsx

```diff
@@ -425,467 +615,539 @@ const LessonEdit = (props: {
     }`; //January 12, 2022, at 3:45 pm
   }
 
+  const trainAIButtonColor =
+    trainStatus.state !== TrainState.SUCCESS &&
+    trainStatus.state !== TrainState.FAILURE
+      ? "secondary"
+      : trainStatus.state === TrainState.FAILURE
+      ? "error"
+      : !(
+          trainStatus.info &&
+          trainStatus.info?.expectations &&
+          Array.isArray(trainStatus.info?.expectations) &&
+          trainStatus.info.expectations.length > 0
+        )
+      ? "error"
+      : Math.min(...trainStatus.info?.expectations.map((x) => x.accuracy)) >=
+        0.6
+      ? "success"
+      : Math.min(...trainStatus.info?.expectations.map((x) => x.accuracy)) >=
+        0.4
+      ? "warning"
+      : "error";
```

Comment:
what's the spec for this?  the expectation results accuracy shouldn't affect whether the training failed or succeeded.